src/celengine/observer.h

```diff
@@ -259,14 +259,15 @@ class Observer
         double expFactor{ 0.5 };
         double accelTime{ AccelerationTime };
         Eigen::Quaterniond rotation1; // rotation on the CircularOrbit around centerObject
+        Eigen::Quaterniond orientationTransformInverse; // the inverse of orientation transform when the journey starts
 
         Selection centerObject;
 
         TrajectoryType traj{ TrajectoryType::Linear };
     };
 
     void gotoJourney(const JourneyParams&);
-    // void setSimulation(Simulation* _sim) { sim = _sim; };
+    void startTraveling();
```

Comment:
Does this need to have public visibility? In this PR it looks like it's only being used by `Observer` internal methods

Reply:
yeah can be private